actions/e2e-check/main.js

```diff
@@ -0,0 +1,89 @@
+const github = require('@actions/github')
+const core = require('@actions/core')
+
+const { context } = github
+
+const githubToken = core.getInput('githubToken')
+const octokit = github.getOctokit(githubToken)
+
+const postMessage = async msg => {
+  await octokit.issues.createComment({
+    issue_number: context.issue.number,
+    owner: context.repo.owner,
+    repo: context.repo.repo,
+    body: `${msg} (E2E status check)`,
+  })
+}
+
+const warn = postMessage
+
+const fail = async msg => {
+  await postMessage(msg)
+  console.error(msg)
+  process.exit(1)
+}
+
+const getComments = async () =>
+  (
+    await octokit.issues.listComments({
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      issue_number: context.payload.number,
+    })
+  ).data
+
+const getLatestE2Ecomment = comments =>
+  comments
+    .filter(c => c.body.trim().indexOf('e2e ') === 0)
```

Comment:
We should explicitly check that it starts with "e2e start" to filter the comments.

```suggestion
    .filter(c => c.body.trim().startsWith('e2e start'))
```

Reply:
This was an interesting one. I had lost track of the adoption of `startsWith`, so I had to check browser compatibility. For reference, the default Babel presets will not transpile this to something else... and IE does not support this at all. However, since our FEWG browser group recently came to the conclusion that we don't support IE, we're good here. 😄 This is awesome though; I definitely hated that I always had to reach for `indexOf` with stuff like this.